### ISSUE TITLE
Nothing happens on Login process after entering the password

### ISSUE THREAD

**Issue by MurzNN** [quexten/goldwarden#189]
I've started the app, unlocked it using pin, filled my Email, pressed Login button, entered my password, the password entering window disappears, and then - nothing happens even after several minutes:
![image](https://github.com/quexten/goldwarden/assets/336662/a8c0e500-595c-4808-831d-0542a8c9ca9d)

The installed version is 0.2.15 from Flatpak. By the way, how to get the exact currently running version number from the app directly? Running something like `flatpak run com.quexten.Goldwarden --version` doesn't work.

What can cause this issue, and how to fix or debug it?

**Comment by quexten** [comment]
What desktop environment / display server (XOrg / Wayland) are you running?

**Comment by MurzNN** [comment]
Operating System: Kubuntu 24.04
KDE Plasma Version: 5.27.11
KDE Frameworks Version: 5.115.0
Qt Version: 5.15.13
Kernel Version: 6.8.0-31-generic (64-bit)
Graphics Platform: X11

**Comment by quexten** [comment]
If the problem is in the GUI i'll postpone triage until after #191

**Comment by quexten** [comment]
> Running something like flatpak run com.quexten.Goldwarden --version doesn't work.
In 0.3.0 in the cli you can do `goldwarden version` and in the GUI the version appears in the about screen.

**Comment by quexten** [comment]
#212
